packages/core/src/components/SupportButton/SupportButton.tsx

```diff
@@ -54,13 +46,42 @@ const useStyles = makeStyles(theme => ({
   },
 }));
 
-export const SupportButton = ({
-  slackChannel = '#backstage',
-  email = [],
-  children,
-}: // plugin,
-PropsWithChildren<Props>) => {
-  // TODO: get plugin manifest with hook
+const SupportIcon = ({ Icon }: { Icon: IconComponent }) => <Icon />;
```

Comment:
is this wrapper needed? 🤔

Reply:
I re-worked this a little bit, but yes it was. Since `getSystemIcon` returns a component type there needs to be a factory component to be used in JSX. Perhaps if this is common usage, then a shared SystemIcon component might be worth creating in an another PR and then perhaps refactoring this + EntityLinksCard to use that maybe.

Reply:
If it's worth it. But this is fine for the foreseeable future then